commands/optional_address.go

```diff
@@ -0,0 +1,17 @@
+package commands
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/filecoin-project/go-filecoin/address"
+)
+
+func optionalAddr(o interface{}) (ret address.Address, err error) {
```

Comment:
Can this be inlined? Do we call it from outside from_address? I don't like having a file per helper function.

Reply:
This is used on lines 374 & 413 of `commands/miner.go` as well as line 150 of `commands/payment_channel.go` for addresses other than the `fromAddr`.